https://github.com/the-tcpdump-group/tcpdump/pull/1068

From 54278acb038f0d16ed75cdddb35fd2813a7cdcef Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Thu, 27 Jul 2023 08:31:53 +0100
Subject: [PATCH] configure.ac: use AC_SYS_LARGEFILE

This enables 64-bit off_t where it's opt-in (e.g. glibc) on 32-bit platforms.

Bug: https://bugs.gentoo.org/911176
Signed-off-by: Sam James <sam@gentoo.org>
--- a/configure.ac
+++ b/configure.ac
@@ -30,7 +30,7 @@ if test "$ac_cv_prog_cc_c99" = "no"; then
 fi
 AC_LBL_C_INIT(V_CCOPT, V_INCLS)
 AC_C_INLINE
-
+AC_SYS_LARGEFILE
 AC_CHECK_HEADERS(fcntl.h rpc/rpc.h rpc/rpcent.h net/if.h)
 
 case "$host_os" in
